Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+  browser_family: Unknown
+
+-
+  user_agent: 'sp-agent ETSfy24nuLJtoscVI1I0Fw'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Samsung Podcasts
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'sp-agent vQAsjA'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Samsung Podcasts
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'ServeStream Dynamo/1.0'
+  os: [ ]
+  client:
+    type: mobile app
+    name: ServeStream
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'ServeStream'
+  os: [ ]
+  client:
+    type: mobile app
+    name: ServeStream
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Shadow/45 CFNetwork/1335.0.3.1 Darwin/21.6.0'
+  os:
+    name: iOS
+    version: "15.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: Shadow
+    version: "45"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Shadowrocket/1858 CFNetwork/1331.0.7 Darwin/21.4.0'
+  os:
+    name: iOS
+    version: "15.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Shadow
+    version: "1858"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Shadow/1 CFNetwork/1402.0.8 Darwin/22.2.0'
+  os:
+    name: iOS
+    version: "16.2"
+    platform: ""
+  client:
+    type: mobile app
+    name: Shadow
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'SiriusXM/1680216748 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: SiriusXM
+    version: "1680216748"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Snipd/2.2.52 CFNetwork/1410.0.3 Darwin/22.6.0 (iPhone iOS/16.6)'
+  os:
+    name: iOS
+    version: "16.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: Snipd
+    version: "2.2.52"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Snipd/90 CFNetwork/1329 Darwin/21.3.0'
+  os:
+    name: iOS
+    version: "15.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: Snipd
+    version: "90"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Sodes/1657 CFNetwork/1098.6 Darwin/19.0.0'
+  os:
+    name: iOS
+    version: "13.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Sodes
```

Comment:
I believe this and the entry below refer to https://apps.apple.com/us/app/sodes/id1321953436 and it looks like the official name is `’sodes`. I believe this name needs to be used

Reply:
I had no idea what this could've been. Thanks.